storage/src/qmdb/benches/variable/init.rs

```diff
@@ -40,21 +40,18 @@ fn bench_variable_init(c: &mut Criterion) {
                             let mut db =
                                 gen_random_kv(db, elements, operations, COMMIT_FREQUENCY).await;
                             db.prune(db.inactivity_floor_loc()).await.unwrap();
-                            db.close().await.unwrap();
```

Comment:
I don't think the call to close is a meaningful part of the benchmark so I removed it